tests/playwright/config/sanity.config.js

```diff
@@ -10,9 +10,10 @@ const config = {
 	retries: 1,
 	expect: {
 		timeout: 5 * 1000, // 5 seconds
+		toMatchSnapshot: { maxDiffPixelRatio: 0.03 },
```

Comment:
I have copied this method, from the latest update to the Pro version: https://github.com/elementor/elementor-pro/blob/84b5a8fdedff12c5bc403b1ac2964e72ca1ba6ba/tests/playwright/config/sanity.config.js#L14.